AppSettings/README.md

```diff
@@ -18,39 +18,41 @@ This will prevent code changes of how you load settings from impacting the rest
 
 Your settings class can look as simple as this:
 
-	class MySettings : SingletonSettings<MySettings>
-	{
-		[AppSetting]
-		public int MyIntValue { get; set; }
+```csharp
```

Comment:
I don't know why we didn't have a code block here or below, but I updated this readme to include them.